src/offchainapi/storage.py

```diff
@@ -231,20 +108,15 @@ def atomic_writes(self):
         return self
 
     def __enter__(self):
-        self.rlock.acquire()
```

Comment:
Do you want to just delete the whole context manager?

Reply:
done